components/user/themes/UserTabs.js

```diff
@@ -1,6 +1,4 @@
-import { useState } from "react";
-import { usePathname, useSearchParams } from "next/navigation";
-
+import { useSearchParams } from "next/navigation";
```

Comment:
`next/navigation` is for use only with the `app` directory.
Since we use the `pages` directory, `router.query` should be used.